src/input/input.styles.scss

```diff
@@ -25,6 +25,18 @@
     }
   }
 
+  .msg {
```

Comment:
I would rename the classe `errorMsg` or something else.
But not `msg` alone, as this doesn't evoke the error nature of the message.

Reply:
:+1: